portfolio/src/main/webapp/index.html

```diff
@@ -101,13 +101,13 @@ <h3>
             
           <div class="courses">
             <ul>
-              <li><span>15-440</span> Distributed Systems</li>
-              <li><span>15-213</span> Introduction to Computer Systems</li>
-              <li><span>05-391</span> Designing Human Centered Software</li>
+              <li><span class ="courseNum">15-440</span> Distributed Systems</li>
```

Comment:
Google CSS style guide asks that CSS identifiers (such as class names) as hyphen separated e.g. course-num instead of underscore_case or camelCase.